scripts/docker/hub/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+echo "#################################################################################"
+echo "THIS IMAGE IS DEPRECATED."
+echo ""
+echo "At some point in the future we will stop pushing new versions of 'parity/parity'."
+echo "Please use 'parity/ethereum' image instead, it is the new canonical location for "
+echo "'parity/parity' images."
+echo ""
+echo "We maintain the same set of tags there, so just replacing the image name"
+echo "everywhere would be safe."
+echo "#################################################################################"
+
+/bin/parity $@
```

Comment:
the `entrypoint.sh` was obsoleted before, why bring it back?
--- and if wasn't it `exec /bin/parity $@` to make parity the main process of the image?

Reply:
Why is the image now called `ethereum` instead of `parity-ethereum` like the repository?

Reply:
Concerning my first comment, afair the entrypoint script was introduced to have a variable binary. Like this there is really no benefit.

Reply:
- just to send this deprecation message
- thanks, I'll check it then
- we decided with @kirushik that why not, it's still in Parity's repo, which makes it `parity/ethereum`
- since `COPY` instruction in the Dockerfile copies the concrete file, `$@` just take the arguments. Is there a way to do it better?

Reply:
That's the original `entrypoint.sh`:
https://github.com/paritytech/parity-ethereum/compare/v2.3.7...master#diff-c346274d88c3013298b1e18cbb88a149

Reply:
ah, you were asking for a better way of printing this message. well then still you can use `exec`.

Reply:
>     * we decided with @kirushik that why not, it's still in Parity's repo, which makes it `parity/ethereum`

so the repository `parity/parity` was renamed to `parity/parity-ethereum` because that was the new name of the software. what is the reason not to do the same for the dockerfile?

Reply:
fixed and simplified, thanks.

Reply:
It was made to be more clear. `ethereum` instead of `parity-ethereum` is only because it stays in `parity` docker namespace.
It will be better than `parity/parity` since we age going to publish more clients there: bitcoin, zcash and whatnot.